.github/workflows/release.yml

```diff
@@ -67,4 +65,5 @@ jobs:
           KEY_ALIAS: ${{ secrets.KEY_ALIAS }}
           KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
           KEY_STORE_PASSWORD: ${{ secrets.KEY_STORE_PASSWORD }}
+          google_json: ${{ secrets.PLAYSTORE_JSON }}
```

Comment:
Either we use "google_json" or "playstore_json". But mixing both names is a bad idea. What is the best?

Reply:
@kelson42 , We have set the ```json``` as ```PLAYSTORE_JSON``` in secrets , so for better naming we should use ```PLAYSTORE_JSON``` for environment variable like we are doing for other environment variables.

Reply:
Ok, then please rename what has to be renamed. Please share as well the name of the secrets which were used in the pas and are not used anymore, so I can remove them.

Reply:
> Ok, then please rename what has to be renamed.

Done.

> Please share as well the name of the secrets which were used in the pas and are not used anymore, so I can remove them.

Earlier we are using ```google_json``` secret to get play publisher credentials but now we are using ```PLAYSTORE_JSON``` to get the details.